bundles/org.platformio.eclipse.ide.workspace/META-INF/MANIFEST.MF

```diff
@@ -4,5 +4,18 @@ Bundle-Name: %Bundle-Name
 Bundle-SymbolicName: org.platformio.eclipse.ide.workspace
 Bundle-Version: 0.1.0.qualifier
 Bundle-Vendor: %Bundle-Vendor
-Automatic-Module-Name: org.platformio.eclipse.ide.workspace
+Automatic-Module-Name: org.platformio.eclipse.ide.workbench
 Bundle-RequiredExecutionEnvironment: JavaSE-11
+Require-Bundle: org.eclipse.core.runtime;bundle-version="0.0.0",
+ org.eclipse.ui;bundle-version="0.0.0",
+ org.eclipse.ui.workbench;bundle-version="0.0.0",
+ org.platformio.eclipse.ide.home.core;bundle-version="0.0.0",
+ org.platformio.eclipse.ide.ui;bundle-version="0.0.0",
```

Comment:
This bundle should be renamed to "org.platformio.eclipse.ide.workbench" and "open file" should go there